spec/lib/rbac/filterer_spec.rb

```diff
@@ -1985,7 +1985,15 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
               - IS NOT EMPTY:
                   field: Vm-name
             ")
-            expect { described_class.search(:class => "Vm", :filter => exp, :user => user, :order => "vms.name desc") }.not_to raise_error
+            expect do
+              described_class.search(:class        => "VmOrTemplate",
```

Comment:
Can we add a more relevant spec(s) related to the BZ instead changing a spec that was designed to test something else?